packages/amplify-migration/src/command-handlers.ts

```
@@ -214,6 +214,35 @@ const unsupportedCategories = (): Map<string, string> => {
   return unsupportedCategoriesList;
 };
 
+export async function updateAmplifyYmlFile(amplifyClient: AmplifyClient, appId: string) {
+  const rootDir = pathManager.findProjectRoot();
+  assert(rootDir);
+  const amplifyYmlPath = path.join(rootDir, 'amplify.yml');
+
+  try {
+    // Read the content of amplify.yml file if it exists
+    await fs.access(amplifyYmlPath);
+    const amplifyYmlContent = await fs.readFile(amplifyYmlPath, 'utf-8');
+
+    await writeToAmplifyYmlFile(amplifyYmlPath, amplifyYmlContent);
+  } catch (error) {
+    // If amplify.yml file doesn't exist, make a getApp call to get buildSpec
```

can we look for a not found error related to `amplify.yml` and have the fallback logic if true? Else we can throw the original error

---

Done